docs/00-Getting-Started.md

```diff
@@ -134,3 +135,5 @@ var myChart = new Chart(ctx, {
 ```
 
 It's that easy to get started using Chart.js! From here you can explore the many options that can help you customise your charts with scales, tooltips, labels, colors, custom actions, and much more.
+
+There are many examples of Chart.js that are available in the [samples folder](https://github.com/chartjs/Chart.js/tree/master/samples) of the GitHub repository.
```

Comment:
Since samples in the repo don't work without a `gulp build` first (and because we encourage users to not clone/download the repository), it may be better to write about the `Chart.js.zip` attached to releases containing built examples in the `samples/` folder.